optuna/multi_objective/visualization/_pareto_front.py

```diff
@@ -0,0 +1,121 @@
+import json
+from typing import List
+from typing import Optional
+
+from optuna._experimental import experimental
+from optuna.multi_objective.study import MultiObjectiveStudy
+from optuna.multi_objective.trial import FrozenMultiObjectiveTrial
+from optuna.visualization.utils import _check_plotly_availability
+from optuna.visualization.utils import is_available
+
+if is_available():
+    from optuna.visualization.plotly_imports import go
+
+
+@experimental("1.5.0")
+def plot_pareto_front(
+    study: MultiObjectiveStudy, names: Optional[List[str]] = None
+) -> "go.Figure":
+    """Plot the pareto front of a study.
+
+    Example:
+
+        The following code snippet shows how to plot the pareto front of a study.
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+               x = trial.suggest_float("x", 0, 5)
+               y = trial.suggest_float("y", 0, 3)
+
+               v0 = (4 * x) ** 2 + (4 * y) ** 2
+               v1 = (x - 5) ** 2 + (y - 5) ** 2
+               return v0, v1
+
+            study = optuna.multi_objective.create_study(["minimize", "minimize"])
+            study.optimize(objective, n_trials=50)
+
+            optuna.multi_objective.visualization.plot_pareto_front(study)
+
+        .. raw:: html
+
+            <iframe src="../../_static/plot_pareto_front.html" width="100%" height="500px"
+            frameborder="0"></iframe>
+
+    Args:
+        study:
+            A :class:`~optuna.study.Study` object whose trials are plotted for their objective
```

Comment:
Ops, should it be `~optuna.multi_objective.study.MultiObjectiveStudy`?

Reply:
Oh..., thank you for the catch!
Fixed: https://github.com/optuna/optuna/pull/1303/commits/a1f2fd60dcb12b3a96b4eec17e2389bdd9c6f799